shared/src/main/scala/tastyquery/ast/Symbols.scala

```diff
@@ -158,8 +169,24 @@ object Symbols {
 
     override lazy val thisType: Type = PackageRef(this)
 
+    override def getDecl(name: Name)(using BaseContext): Option[Symbol] =
+      super.getDecl(name).orElse {
```

Comment:
minor but this is going to call `summon[BaseContext].classloader.scanClass(this)` unnecessarily

Reply:
Ah, I had not realized that `PackageClassSymbol` extends `ClassSymbol`. I changed it to directly call `getDeclInternal` instead.